### DISCUSSION TITLE
Spracingf3

### DISCUSSION THREAD

**Discussion by obermario** [iNavFlight/inav#9044] [General]
Tried to revive two spracing f3 boards. However when trying to flash with multiple configurator and firmware versions I always get the following error:
STM32 - timed out, programming: FAILED

My solution was to download the inav hex file for the spracingf3 and Flash it with the Cleanflight configurator chrome app.
Betaflight Configurator didn't even start to flash.

Just wanted to leave this here if someone encounters the same problem.

**Comment by tsouth1** [comment]
Hello, where can I download this inav hex file for the spracingf3 fc?

**Reply by OptimusTi** [reply]
If there's one it's going to be in the Releases page: https://github.com/iNavFlight/inav/releases
